mcs/build/profiles/monodroid.make

```diff
@@ -26,6 +26,7 @@ PROFILE_MCS_FLAGS = \
 	-d:MONODROID \
 	-d:ANDROID \
 	-d:NETSTANDARD \
+	-d:NO_CONSOLE \
```

Comment:
this won't work, the PROFILE_MCS_FLAGS only get passed to the C# compiler so they have no effect on the Makefile. You need to set `NO_CONSOLE=yes` further down as part of the Makefile instead.